### ISSUE TITLE
Not able deploy using the run.bat

### ISSUE THREAD

**Issue by manojkharol** [microsoft/Learn-LTI#86]
I ran the run.bat script. I was able to successfully login my azure account and later it asked me for my subscription name. I provided the same but there was an error of Invalid subscription name (screenshot attached). After this, it asked for the region and I mentioned eastus2. Later there is an error saying Insufficient Privilege (screenshot attached). 
![image](https://user-images.githubusercontent.com/46344301/97327266-d86d3d00-189a-11eb-80fd-31e221db23b5.png)
![image](https://user-images.githubusercontent.com/46344301/97327460-fc308300-189a-11eb-80b5-9f89701e985c.png)

**Comment by manojkharol** [comment]
The issue is resolved by commenting the validation code from line 96 to 99 in Deploy.ps1 file and adding owner in the Azure Portal.
![image](https://user-images.githubusercontent.com/46344301/97555049-a1fc0300-19fd-11eb-9ad5-a3e2ba49f05c.png). Thanks for the help Aditya! 

However, there is a new issue while Creating Resources in Azure.
![image](https://user-images.githubusercontent.com/46344301/97555292-f4d5ba80-19fd-11eb-9295-3367ffb7797c.png)
Deployment failed. Correlation ID: ab846c6a-0ab2-4fef-9b8d-01a80a577074. {
  "status": "failed",
  "error": {
    "code": "ResourceDeploymentFailure",
    "message": "The resource operation completed with terminal provisioning state 'failed'.",
    "details": [
      {
        "code": "DeploymentScriptError",
        "message": "The provided script failed with multiple errors. First error:\r\nERROR: Please run 'az login' to setup account.. Please refer to https://aka.ms/DeploymentScriptsTroubleshoot for more deployment script information.",
        "details": [
          {
            "code": "DeploymentScriptError",
            "message": "ERROR: Please run 'az login' to setup account."
          },
          {
            "code": "DeploymentScriptError",
            "message": "ERROR: Please run 'az login' to setup account."
          }
        ]
      }
    ]
  }
}

**Comment by adbindal** [comment]
@manojkharol, #87 should take care of the `az login` issue that you are currently facing. We've also updated the deployment scripts to start accepting both Subscription ID and Name as user input (see #88). Hope, these changes, once merged, should unblock the deployment on your end.

**Comment by leestott** [comment]
@manojkharol can you confirm this change has fixed your issue.

**Comment by adbindal** [comment]
I had got a chance to sync-up with @manojkharol on this and the above issue has been resolved for them. The team is currently in the process of onboarding the tool with their LMS.
